<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>811</width>
      <height>311</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Categories</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Job Success</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>PerHourCharges</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Offers consultations</string>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Location</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Earnings</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ratings</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="start_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resume_btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Resume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_btn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sorting_CB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Merge Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hybrid merge sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Selection Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Insertion Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bubble Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Heap sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cycle Sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Count Sort</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="searching_CB">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Contain</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>End With</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Start With</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>And</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Or</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Not</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="sorting_btn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Sorting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searching_btn">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Searching</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>771</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Consumed In Sorting</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>811</width>
      <height>621</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="time_Display">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Text Label</string>
     </property>
    </widget>
    <widget class="QPushButton" name="scraper_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Scraper Page</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CrossButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MinimizeButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimize</string>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>tableWidget</zorder>
   <zorder>start_btn</zorder>
   <zorder>resume_btn</zorder>
   <zorder>stop_btn</zorder>
   <zorder>sorting_CB</zorder>
   <zorder>searching_CB</zorder>
   <zorder>sorting_btn</zorder>
   <zorder>searching_btn</zorder>
   <zorder>progressBar</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
